source/WorkingPractices/macros.rst

```diff
@@ -92,6 +92,10 @@ The organisation of LFRic metadata is different from other repositories (UM + Ju
 
 To solve this, macros in LFRic Apps are applied using a wrapper script which will read the added macros and combine them into the versions.py files for the apps where that metadata is imported. Therefore when adding macros, the macro should be added in the versions.py file in the same metadata directory as the metadata change being made. It will then be shared as appropriate by the ``apply_macros.py`` script.
 
+.. tip::
+
+    The macro will only end up in versions.py files for metadata that is directly imported by a rose-stem app. Therefore if adding to eg. Science/gungho, the macro will be deleted from that file by the script.
```

Comment:
I wonder if a final sentence on here to drive home the "make sure you're ready for things to be deleted" message?